azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/springcloud/AbstractMojoBase.java

```diff
@@ -186,6 +192,19 @@ protected void initTelemetry() {
         traceConfiguration(this.getConfiguration());
     }
 
+    protected void printCredentialDescription(Account account) {
```

Comment:
I can see another `printCredentialDescription` upper.
why not move this method into `Account` as its `toString`;
and then `println(account.toString())`;